Jenkinsfile

```diff
@@ -17,33 +17,44 @@ node{
                 checkout scm
             }
         }
-       
+        
         stage('Lint'){
             linters.each {
                 linter_jobs["${it}"] = {
                     node('linux'){
                         checkout scm
-                        def pip_cache = "${env.WORKSPACE}/${env.PIP_CACHE}"
-                        sh("./tests/scripts/docker/${it}.sh ${pip_cache}")
+                        try{ sh 'docker stop $(docker ps -q -a)' }catch(e){}
+                        try{ sh 'docker rm -v $(docker ps -a -q)' }catch(e){}
+                        sh("./tests/scripts/docker/${it}.sh")
                     }
                 }
             }
             parallel linter_jobs
         }
+
+        stage('Docs'){
```

Comment:
I like that the doc build is not a separate stage.

Reply:
is the `not` a typo and you meant `now`?

Reply:
yep, typo. sorry.